security/certificate_types_descriptions/service-ca-certificates.adoc

```diff
@@ -27,6 +27,11 @@ The service CA expiration of 26 months is longer than the expected upgrade inter
 A manually-rotated service CA does not maintain trust with the previous service CA. You might experience a temporary service disruption until the pods in the cluster are restarted, which ensures that pods are using service serving certificates issued by the new service CA.
 ====
 
+[IMPORTANT]
+====
+Applications that use a different truststore that have a dependency on `service-ca` certificate, such as JavaKeyStore, need to be capable of accepting new CA certifications dynamically into it's truststore. Otherwise, when automated rotation occurs, the application pods will require a restart.
```

Comment:
it's x its
certification x certificate


> Applications that use a different truststore that have a dependency

I don't understand that sentence. Truststore? Different from what?

Reply:
I think that I've fixed your concerns. PTAL.